cli/src/config.rs

```diff
@@ -29,6 +29,8 @@ pub struct Configuration {
     pub private_key: PrivateKey,
     /// Disable coloring of the backtrace and error report on panic.
     pub disable_panic_terminal_colors: bool,
+    /// Iroha will shutdown on any panic if this option is set to `true`.
+    pub enable_shutdown_on_panic: bool,
```

Comment:
maybe just `shutdown_on_panic`?

Reply:
Yeah, it's better. I just followed above convention (`disable_panic_terminal_colors`)